docs/source/contents/5_technical_metadata_properties.rst

```diff
@@ -0,0 +1,245 @@
+####################
+Technical Properties
+####################
+
+*****
+About
+*****
+
+While our MODS to RDF mapping aims to address how descriptive properties should be mapped in our next system, this
+section aims to explain how to map existing technical metadata properties our metadata properties that we need going
```

Comment:
I think you're missing a "to"? Should it say "map existing technical metadata properties to our metadata properties"?